includes/admin/class-wc-stripe-onboarding-controller.php

```diff
@@ -32,6 +32,8 @@ public function admin_scripts() {
 				'dependencies' => [],
 				'version'      => WC_STRIPE_VERSION,
 			];
+		$style_path = 'build/style-additional_methods_setup.css';
```

Comment:
I wonder if we could try to adopt a single name for this functionality since right now it's being called both "additional methods setup" and "onboarding wizard" which can be confusing. What do you think about making everything "onboarding wizard" for clarity? Meaning the entrypoint in webpack.config.js and the folder name?

Reply:
Renaming this to `upe_onboarding_wizard` so it follows the naming here: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/5b6f881e4d904fd64038f772e8d55c6b83d06b16/webpack.config.js#L25-L28

Reply:
Renamed all instances of 
- `additional_methods_setup` to `upe_onboarding_wizard`
- `additional-methods-setup` to `upe-onboarding-wizard`

@waclawjacek , i renamed the folder and changed the filename (https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1735/commits/89afebcee6495bc3d324bc1169064586619157a3).